frontend/src/components/ShootTickets/TicketComment.vue

```diff
@@ -94,6 +94,30 @@ export default {
     word-wrap: break-word;
     /* does not work with firefox */
     word-break: break-word;
+
```

Comment:
why is this css now required?

Reply:
it's not required, however I overwrote the font-size of the headings and adapted them to the github sizes. Previously, I felt they were a bit too large

Reply:
okay, I just wanted to confirm because it has nothing to do with the rest of the PR